packages/react-components/src/components/styles/TogetherMode.styles.ts

```diff
@@ -0,0 +1,171 @@
+/**
+ * Sets the seating position for a participant in Together Mode.
+ *
+ * @param seatingPosition - The seating position information.
+ * @returns The style object for the seating position.
+ */
+export function setParticipantSeatingPosition(
+  seatingPosition: VideoGalleryTogetherModeSeatingInfo
+): TogetherModeParticipantSeatPosition {
+  return {
+    width: _pxToRem(seatingPosition.width),
+    height: _pxToRem(seatingPosition.height),
+    left: _pxToRem(seatingPosition.left),
+    top: _pxToRem(seatingPosition.top)
+  };
+}
+
+/* @conditional-compile-remove(together-mode) */
+/**
+ * Return a style bucket based on the number of active sprites.
+ * For example, the first three reactions should appear at maximum
+ * height, width, and opacity.
+ * @private
+ */
+export function setTogetherModeSeatPositionStyle(
+  seatingPosition: VideoGalleryTogetherModeSeatingInfo
+): TogetherModeSeatStyle {
+  return {
+    seatPosition: setParticipantSeatingPosition(seatingPosition)
+  };
+}
+
+/* @conditional-compile-remove(together-mode) */
+/**
+ * Generates the overlay style for a participant in Together Mode.
+ *
+ * @param seatingPosition - The seating position information.
+ * @returns The style object for the participant overlay.
+ */
+export function getTogetherModeParticipantOverlayStyle(
+  seatingPositionStyle: TogetherModeSeatStyle
+): React.CSSProperties {
+  return {
+    ...seatingPositionStyle.seatPosition,
+    position: 'absolute'
+  };
+}
+
+// Function to map a value from one range to another
+const mapRange = (value: number, inMin: number, inMax: number, outMin: number, outMax: number): number => {
+  return outMin + ((value - inMin) * (outMax - outMin)) / (inMax - inMin);
+};
+
+/**
+ * Calculate the scaled size based on width and height.
+ *
+ * @param width - The width of the element.
+ * @param height - The height of the element.
+ * @returns The scaled size.
+ */
+export const calculateScaledSize = (width: number, height: number): number => {
+  const maxSize = 600;
+  const minSize = 200;
+  const minScaledSize = 35;
+  const maxScaledSize = 70;
+
+  // Use width or height to determine scaling factor
+  const size = Math.min(width, height);
+
+  // Map the size to the desired range
+  return mapRange(size, minSize, maxSize, minScaledSize, maxScaledSize);
+};
+
+/* @conditional-compile-remove(together-mode) */
+/**
+ * @private
+ */
+export const togetherModeStreamRootStyle: IStackStyles = {
+  root: {
+    width: '100%',
+    height: '100%',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    position: 'absolute',
+    top: 0,
+    left: 0
+  }
+};
+
+/* @conditional-compile-remove(together-mode) */
+/**
+ * @private
+ */
+export const togetherModeIconStyle = (): React.CSSProperties => {
+  return {
+    width: _pxToRem(20),
+    flexShrink: 0
+  };
+};
+
+/* @conditional-compile-remove(together-mode) */
+/**
+ * @private
+ */
+export const togetherModeParticipantStatusContainer = (
```

Comment:
Can we have a majority of these styles in a separate style page. 
And then just have a merge styles instead for specific background color, radius?

Same thing for the togetherModeParticipantDisplayName